time timestamp
int32 track_id
float32 left
float32 top
float32 right
float32 bottom